pkg/provisioners/channel_util.go

```diff
@@ -63,19 +63,35 @@ func RemoveFinalizer(o metav1.Object, finalizerName string) {
 	o.SetFinalizers(finalizers.List())
 }
 
-func CreateK8sService(ctx context.Context, client runtimeClient.Client, c *eventingv1alpha1.Channel) (*corev1.Service, error) {
+type k8sServiceOption func(*corev1.Service) error
+
+// ExternalService is a functional option for CreateK8sService to create a K8s service of type ExternalName
```

Comment:
Add a trailing period.

Reply:
I guess I need a bot next time to catch this one. Have repeated this for every PR I have sent so far :P